### ISSUE TITLE
typescript removes necessary commas/semicolons in object literal types w/ --no-semi

### ISSUE THREAD

**Issue by aaronjensen** [prettier/prettier#1942]
```typescript
let foo2: {a: string, b: string}
```

becomes:

```typescript
let foo2: { a: string b: string }
```

which is a parse error.

https://prettier.github.io/prettier/#%7B%22content%22%3A%22let%20foo%3A%20Foo%3C%7Ba%3A%20string%2C%20b%3A%20string%7D%3E%5Cnlet%20foo2%3A%20%7Ba%3A%20string%2C%20b%3A%20string%7D%22%2C%22options%22%3A%7B%22printWidth%22%3A80%2C%22tabWidth%22%3A2%2C%22singleQuote%22%3Afalse%2C%22trailingComma%22%3A%22none%22%2C%22bracketSpacing%22%3Atrue%2C%22jsxBracketSameLine%22%3Afalse%2C%22parser%22%3A%22typescript%22%2C%22semi%22%3Afalse%2C%22useTabs%22%3Afalse%2C%22doc%22%3Afalse%7D%7D

**Comment by vjeux** [comment]
I just published 1.4.2 with this fix.

**Comment by aaronjensen** [comment]
That was quick, thanks!
